Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -47,18 +51,20 @@ - (void)trackApplicationUIKitNotifications {
                                                     SentryBreadcrumb *crumb = [[SentryBreadcrumb alloc] initWithLevel:kSentrySeverityWarning category:@"Device"];
                                                     crumb.type = @"system";
                                                     crumb.message = @"Memory Warning";
-                                                    [SentryClient.sharedClient.breadcrumbs addBreadcrumb:crumb];
+                                                    [SentrySDK.currentHub addBreadcrumb:crumb];
                                                 }];
 #endif
 }
      
 - (void)addEnabledCrumb {
-    if (nil != SentryClient.sharedClient) {
-        SentryBreadcrumb *crumb = [[SentryBreadcrumb alloc] initWithLevel:kSentrySeverityInfo category:@"started"];
-        crumb.type = @"debug";
-        crumb.message = @"Breadcrumb Tracking";
-        [SentryClient.sharedClient.breadcrumbs addBreadcrumb:crumb];
-    }
+    SentryBreadcrumb *crumb = [[SentryBreadcrumb alloc] initWithLevel:kSentrySeverityInfo category:@"started"];
+    crumb.type = @"debug";
+    crumb.message = @"Breadcrumb Tracking";
+
+    // TODO(fetzig): don't know if configureScope is the right way to do this.
+    [SentrySDK.currentHub configureScope:^(SentryScope * _Nonnull scope) {
```

Comment:
`don't know if configureScope is the right way to do this.`
Generally yes, but why not just `[SentrySDK addBreadcrumb:crumb];`?

Reply:
so true... replaced that somewhere else as well.